recipes/acts/recipe.yaml

```
@@ -0,0 +1,47 @@
+context:
+  name: acts
+  version: "41.1.0"
+
+package:
+  name: ${{ name|lower }}-core
+  version: ${{ version }}
```

```suggestion
recipe:
  # Have top level name be unique from any outputs
  # c.f. https://github.com/conda-forge/conda-forge.github.io/blob/abfc33db28c35e9a8f6b719d0021768f0d5d06be/docs/maintainer/knowledge_base.md?plain=1#L1749
  name: ${{ name }}-suite
  version: ${{ version }}
```

Look at https://github.com/conda-forge/fastjet-cxx-feedstock/blob/120bb1c08fb582e67e50851421d01bea47e176a2/recipe/recipe.yaml for how a multi-output recipe is setup and also read this section of the `rattler-build` docs: https://rattler.build/latest/reference/recipe_file/#outputs-section. For these kinds of recipes, the top level gets defined as `recipe` and then different outputs get defined with most of the rest of the recipe indented under them. Look those over and then revise the rest of the recipe to follow this multiple output format, even though we're only starting with one output (`acts-core`).